### PR TITLE
Re-enable conmon-rs attach test

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind ci

#### What this PR does / why we need it:
The test case should be fixed with the recent conmonrs version, so we can re-enable it.

#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by saschagrunert** [comment]
@haircommander tests are now green, do we wanna wait for the next Podman and containers/* releases or are you happy to vendor in the latest `main` of them?

File: go.mod [hunk 1]
```diff
@@ -15,14 +15,14 @@ require (
...
+	github.com/containers/common v0.48.1-0.20220712112137-79796831707b
 	github.com/containers/conmon v2.0.20+incompatible
-	github.com/containers/conmon-rs v0.0.0-20220609131637-836ba11bf0aa
-	github.com/containers/image/v5 v5.21.1
+	github.com/containers/conmon-rs v0.0.0-20220713152459-5f3451efdbbd
+	github.com/containers/image/v5 v5.21.2-0.20220712113758-29aec5f7bbbf
 	github.com/containers/ocicrypt v1.1.5
-	github.com/containers/podman/v4 v4.1.1
-	github.com/containers/storage v1.41.0
+	github.com/containers/podman/v4 v4.0.0-rc2.0.20220713192916-5c3eec55895c
```
Comment (haircommander) [inline go.mod:24]: hm this looks like a regression

Comment (saschagrunert) [reply go.mod]: Fixed

**Comment by saschagrunert** [comment]
@haircommander PTAL

**Pull request merged**